proxy/src/coco/source.rs

```diff
@@ -8,8 +8,17 @@ use radicle_surf::{
     diff, file_system,
     vcs::git::{self, git2, BranchName, Browser},
 };
+use syntect::parsing::SyntaxSet;
 
 use crate::error;
+use crate::session::settings::Theme;
+
+lazy_static::lazy_static! {
+    // The syntax set is slow to load (~30ms), so we make sure to only load it once.
+    // It _will_ affect the latency of the first request that uses syntax highlighting,
+    // but this is acceptable for now.
```

Comment:
Is there no way to warm it up?

Reply:
Not unless we run it in some other thread or something..